docs/DeveloperGuide.md

```diff
@@ -514,7 +514,7 @@ testers are expected to do more *exploratory* testing.
 
 </div>
 
-### Launch and shutdown
+### Launch and Shutdown
 
 1. Initial launch
```

Comment:
the numbering for this section needs to be updated as well

Reply:
how so?

Reply:
Do you mean to follow the Use Case style numbering?

Reply:
nope haha just that some parts numberings are not incrementing